x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -276,7 +273,15 @@ const UnstyledProcessEventDot = React.memo(
      * Statistics for the number of related events and alerts for this process node
      */
     relatedEventsStatsForProcess?: ResolverNodeStats;
+
+    /**
+     * The time (unix epoch) at render.
+     */
+    timeAtRender: number;
   }) => {
+    // This should be unique to each instance of Resolver
+    const htmlIDPrefix = 'resolver';
```

Comment:
@kqualters-elastic we need use the documentLocationID here. that way we can have unique IDs.